style.css

```diff
@@ -18,9 +18,36 @@ h3, p {
   font-family: "Skia", "Open Sans", sans-serif
 }
 
+.body-image{
+  width: 30%;
+  min-width: 200px;
+  display: flex;
+  float: left;
+}
+
+
+.body-table{
+  width: 40%;
```

Comment:
Probably shouldn't use custom styling for dimensions. Let Bootstrap grid take care of that so it remains mobile friendly.